src/NexusMods.App.UI/WorkspaceSystem/Panel/PanelView.axaml.cs

```diff
@@ -32,6 +34,19 @@ public PanelView()
                 .Subscribe()
                 .DisposeWith(disposables);
 
+            // panel selection
+            this.AddDisposableHandler(PointerEnteredEvent, (_, _) =>
+            {
+                if (ViewModel is not null) ViewModel.IsSelected = true;
+            }, routes: RoutingStrategies.Direct | RoutingStrategies.Bubble, handledEventsToo: true).DisposeWith(disposables);
```

Comment:
This changes selection on pointer entered hover, the desired behavior is selection to change on interaction with contents.

Reply:
Right, forgot to change that.